nestkernel/connection.h

```diff
@@ -401,6 +403,7 @@ SecondaryEvent*
 Connection< targetidentifierT >::get_secondary_event()
 {
   assert( false );
+  std::abort();
```

Comment:
The assertion there is indeed plain wrong. So we can also stick to only have `std::abort()` there.
```suggestion
  std::abort();
```

Reply:
I think that the assertion makes more sense here than abort: this method should simply never been called in the base class. And the `assert()` can be compiled out with `-DNDEBUG`.